### DISCUSSION TITLE
Technical question about sequential numbering of inscriptions

### DISCUSSION THREAD

**Discussion by jmbinette** [ordinals/ord#1644] [Q&A]
If we have 2 inscriptions on the same BTC block, same timestamp.
What determines the sequential inscription # and which one occured first on the blockchain ?

This have a major impact on current BTC collections

**Comment by casey** [comment]
When a miner constructs a candidate block, they are free to include transactions in any order. This order then determines the numbering of inscriptions in those transactions. Eventually we want to allow multiple inscriptions per transaction, which will allow the inscribe to at least make sure that those inscriptions are sequential and in a specific order.

**Comment by jmbinette** [comment]
Hi Casey,

For historical accuracy, wouldn't it be right to use the tx submission timestamp instead as a tiebreaker ?

Since BTC block times are long compared to other chains, technically an inscription 10 minutes later than any other inscription could be inscribed lower with a 1,000sat/vByte fee for example

**Comment by jmbinette** [comment]
BTC projects and Ordinals sequential order are two different things, I agree.

One could watch the unconfirmed tx in the mempool and jump all of them with a high fee. They would have plenty of time.

To me, once a tx is confirmed, no matter the fee paid. If it is confirmed, you are first if the timestamp is the oldest
